projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1139,6 +1139,25 @@
         bottom: 0;
     }
 
+    %igx-grid__td--bool {
+        .igx-grid__td-text {
```

Comment:
You should **never** refer to css class selectors generated using the BEM mixins here. Also, you don't have to modify the text field.